### ISSUE TITLE
Rainbow Effect - Unaffected Top Keys

### ISSUE THREAD

**Issue by MrInterBugs** [zauguin/libx50q#2]
Thought I would give this a play and see if there would be much option for me to create a OpenRGB controller.

Do you know why the media controls and brightness key are unaffected by setting the cycle effect on?

**Comment by zauguin** [comment]
No. IIRC it was different in an older firmware and then changed when they also dropped the built-in mute button.

They are not completely unaffected though, just like the other keys the configured color gets ignored once they are put into cycle mode, it is just replaced with a fixed color instead of varying colors.

**Comment by MrInterBugs** [comment]
No worries thanks for replying so quick. Such a PITA keyboard....
